.github/workflows/checkLicenses.yml

```diff
@@ -15,13 +15,7 @@
 name: Check Licenses
 on:
```

Comment:
do you want just "on: [push, pull_request]"?

How about other workflows? https://github.com/GoogleContainerTools/kpt/issues/2229

Reply:
Changed to just `pull_request` for all workflows bar releasing the tool or deploying the site. None of the workflows that check for a pristine state modify the state of the repo after merging a PR, so this should be sufficient until that's no longer the case.

Reply:
I think you want workflows to run on push. This visually ends up as a green check mark on the branch.

Reply:
Added for the sake of validation.